src/main/java/org/prebid/server/bidder/yahoossp/YahooSSPBidder.java

```diff
@@ -144,6 +166,39 @@ private static Banner modifyBanner(Banner banner) {
                 .build();
     }
 
+    private Regs modifyRegs(Regs regs) {
+        Integer gdpr = regs.getGdpr();
+        String usPrivacy = regs.getUsPrivacy();
+        final String gpp = regs.getGpp();
+        final List<Integer> gppSid = regs.getGppSid();
+
+        if (regs.getExt() != null) {
+            if (gdpr == null) {
+                gdpr = regs.getExt().getGdpr();
+            }
+            if (usPrivacy == null) {
+                usPrivacy = regs.getExt().getUsPrivacy();
+            }
+        }
+
+        final ExtRegs extRegs = ExtRegs.of(gdpr, usPrivacy);
```

Comment:
Create separate method resolveExtRegs()

Reply:
I moved the building of the new regs ext into a separate method.